e2e/js/js-esm/playwright.config.ts

```diff
@@ -0,0 +1,41 @@
+import { defineConfig } from '@playwright/test'
+import { nxE2EPreset } from '@nx/playwright/preset'
+
+import { workspaceRoot } from '@nx/devkit'
+
+// For CI, you may want to set BASE_URL to the deployed application.
+const baseURL = process.env['BASE_URL'] || 'http://127.0.0.1:3000'
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+const nxConfig = nxE2EPreset(__filename, { testDir: './tests' })
+
+export default defineConfig({
+    ...{
+        ...nxConfig,
+        // Only run the webkit browser.
+        projects: nxConfig.projects.filter((p) => p.name === 'webkit'),
+    },
```

Comment:
would recommend double checking that the playwright report is being generated. I had to add an extra `reporter` config option here

Reply:
Otherwise debugging issues that pop up in CI might be annoying

Reply:
done 👍🏼